scripts/setup.sh

```diff
@@ -155,9 +160,11 @@ else
   warn "Development mode on: custom S3 url link."
 fi
 
-echo -e
-echo "- downloading functions source [$FUNCTION_REPOSITORY_RELEASE_URL]"
-wget -q $FUNCTION_REPOSITORY_RELEASE_URL -O $WORKING_DIR/$FUNCTION_ZIP_PACKAGE
+if [[ "$USE_LOCAL_FUNCTION_ZIP" != "Y" ]]; then
```

Comment:
Minor - we could print warning when '--use-local-function-zip' is used, simialr to line 160